autoload/ale/engine.vim

```diff
@@ -177,6 +177,7 @@ function! s:HandleExit(job_id, exit_code) abort
     let l:output = l:job_info.output
     let l:buffer = l:job_info.buffer
     let l:next_chain_index = l:job_info.next_chain_index
+    let l:wd = l:job_info.wd
```

Comment:
No, don't do this. This will mean we have to compute this information for linters even where it isn't relevant. Fix the problem in the GHC linter.